src/core/connectors.js

```diff
@@ -1740,6 +1740,13 @@ const connectors = [{
 	],
 	js: 'connectors/turntable.fm.js',
 	id: 'turntable.fm',
+}, {
+	label: 'Stingray Music',
+	matches: [
+		'*://webplayer.stingray.com/*',
```

Comment:
I would replace this pattern with `*.stingray.com/*` to let the extension mark the main website (stingray.com) as supported.